test/reply-code.test.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
```

Comment:
I don't understand this test file in the context of the issue. How is it testing for valid listen port values?

Reply:
My previous pull request also changed how reply status code is parsed. This is why I had to add this reply related status coersion tests.

Reply:
This is the problem with PRs doing more than a single thing.

Reply:
I agree with you 100%.

Reply:
Can you split the changes of this file to a separate PR?

Reply:
If this is not a blocker for this pull request, I prefer not to do it, but otherwise I'll.